elide-contrib/dropwizard-elide/src/main/java/com/yahoo/elide/contrib/dropwizard/elide/ElideBundle.java

```diff
@@ -130,6 +137,14 @@ public DataStore getDataStore(T configuration, Environment environment) {
         return new HibernateStore(sessionFactory);
     }
 
+    public ImmutableList<Class<?>> getEntities() {
```

Comment:
so that classes that implement `ElideBundle` can get access to the entities. They can build different dataStore out of it